sync.go

```diff
@@ -473,6 +436,17 @@ func (lw *LibWallet) GetBestBlockTimeStamp() int64 {
 	return info.Timestamp
 }
 
-func (lw *LibWallet) GetConnectedPeersCount() int32 {
-	return lw.connectedPeers
+func (mw *MultiWallet) GetConnectedPeersCount() int32 {
```

Comment:
We have a similar `ConnectedPeers` method that's more correctly implemented. This can be removed. Should probably rename `ConnectedPeers` to `GetConnectedPeersCount`